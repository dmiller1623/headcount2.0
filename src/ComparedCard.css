.comparedCard {
  align-items: center;
  background-image: linear-gradient(to top, #fff1eb 0%, #ace0f9 100%);  border: solid 2px black;
  border: solid 2px black;
  border-radius: 15px;
  display: flex;
  flex-direction: column;
  height: 23rem;
  justify-content: space-between;
  margin: .2rem;
  width: 14rem;
}

.comparedName {
  color: black;
  font-size: 1.5rem;
}

.averages {
  color: white;
  font-size: 1.8rem;
  margin: 0rem;
}

.compared {
  color: #694460;
  font-size: 2rem;
}
 p {
  margin: 0;
  padding: 0;
 }