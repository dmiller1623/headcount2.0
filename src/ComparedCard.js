import React from 'react';
import './ComparedCard.css';
import PropTypes from 'prop-types';

const ComparedCard = ( { 
  firstDistrict,
  compared,
  secondDistrict, 
  firstDistrictName, 
  secondDistrictName } ) => {
  return (
    <div className="comparedCard">
      <h2 className="comparedName">{firstDistrictName}</h2>
      <p>&darr;</p>
      <h5 className="averages">&lt;-----{firstDistrict}</h5>
      <h2 className="compared">&lt;--{compared}--&gt;</h2>
      <h5 className="averages">{secondDistrict}------&gt;</h5>
      <p>&uarr;</p>
      <h2 className="comparedName">{secondDistrictName}</h2>
    </div>
  );
};

ComparedCard.propTypes = { 
  firstDistrict: PropTypes.number,
  compared: PropTypes.number,
  secondDistrict: PropTypes.number,
  firstDistrictName: PropTypes.string,
  secondDistrictName: PropTypes.string
};


export default ComparedCard;