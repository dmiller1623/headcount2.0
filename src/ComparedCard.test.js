import React from 'react';
import ReactDOM from 'react-dom';
import ComparedCard from './ComparedCard';
import { shallow, mount } from 'enzyme';

describe('ComparedCard',() => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<ComparedCard />)
    expect(wrapper).toMatchSnapshot()
  })

})