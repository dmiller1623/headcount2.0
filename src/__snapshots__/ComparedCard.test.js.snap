// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComparedCard should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ComparedCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="comparedName"
/>,
        <p>
          ↓
</p>,
        <h5
          className="averages"
>
          &lt;-----
</h5>,
        <h2
          className="compared"
>
          &lt;--
          --&gt;
</h2>,
        <h5
          className="averages"
>
          ------&gt;
</h5>,
        <p>
          ↑
</p>,
        <h2
          className="comparedName"
/>,
      ],
      "className": "comparedCard",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "comparedName",
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "↓",
        },
        "ref": null,
        "rendered": "↓",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "<-----",
            undefined,
          ],
          "className": "averages",
        },
        "ref": null,
        "rendered": Array [
          "<-----",
          undefined,
        ],
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "<--",
            undefined,
            "-->",
          ],
          "className": "compared",
        },
        "ref": null,
        "rendered": Array [
          "<--",
          undefined,
          "-->",
        ],
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            "------>",
          ],
          "className": "averages",
        },
        "ref": null,
        "rendered": Array [
          undefined,
          "------>",
        ],
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "↑",
        },
        "ref": null,
        "rendered": "↑",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "comparedName",
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="comparedName"
/>,
          <p>
            ↓
</p>,
          <h5
            className="averages"
>
            &lt;-----
</h5>,
          <h2
            className="compared"
>
            &lt;--
            --&gt;
</h2>,
          <h5
            className="averages"
>
            ------&gt;
</h5>,
          <p>
            ↑
</p>,
          <h2
            className="comparedName"
/>,
        ],
        "className": "comparedCard",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "comparedName",
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "↓",
          },
          "ref": null,
          "rendered": "↓",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "<-----",
              undefined,
            ],
            "className": "averages",
          },
          "ref": null,
          "rendered": Array [
            "<-----",
            undefined,
          ],
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "<--",
              undefined,
              "-->",
            ],
            "className": "compared",
          },
          "ref": null,
          "rendered": Array [
            "<--",
            undefined,
            "-->",
          ],
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              "------>",
            ],
            "className": "averages",
          },
          "ref": null,
          "rendered": Array [
            undefined,
            "------>",
          ],
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "↑",
          },
          "ref": null,
          "rendered": "↑",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "comparedName",
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
